pages/glossary/distinguishing-styles.md

```diff
@@ -0,0 +1,15 @@
+--
+title: Distinguishing Styles
+key: distinguishing-styles
+unambiguous: true
+objective: true
+--
+
+Two elements have distinguishing styles if any of the following is true:
+
+- they have [distinguishing borders](#distinguishing-borders); or
+- they have [distinguishing box-shadows](#distinguishing-box-shadows); or
+- they have different computed values for the `background-image` property; or
```

Comment:
Does background image not fit under distinguishing content already?

Reply:
Not really. A distinguishing background is not, by itself, an indication that the target is a link (which is the goal of the distinguishing content). But it's a distinguishing style.